src/Symfony/Bundle/DebugBundle/Resources/config/services.xml

```diff
@@ -36,11 +36,6 @@
             <argument>null</argument>
             <argument>%kernel.charset%</argument>
             <argument>0</argument> <!-- flags -->
-            <call method="setDisplayOptions">
-                <argument type="collection">
-                    <argument key="fileLinkFormat" type="service" id="debug.file_link_formatter"></argument>
```

Comment:
@core23 I know. and a community review asked you to change it, but several core team members looking at your PR are preferring the first implementation until now (and other core team members have not said what they prefer).